### DISCUSSION TITLE
Introduction/Interest in Project 1: BeadGroup & RingGroup

### DISCUSSION THREAD

**Discussion by JiafengDu** [MDAnalysis/mdanalysis#4509] [GSoC Discussions]
Hello, just submitted my pre-proposal tonight! Looking forward to hearing back from y'all!

Here is a little bit about me:

I graduated from the University of California, Irvine in 2021 with a degree in math and a degree in physics with a concentration in computational physics. In school, I used COMSOL for thermodynamic simulation for my research project. My relevant courses include Numerical Analysis and Numerical Differential Equations. I worked in healthcare IT as a technical solutions engineer after my undergraduate. 
Now, I am starting my first semester as a master's student at Northeastern University for computer science. This semester, I am taking data structure and algorithm and object-oriented design. In the upcoming summer, I will take advanced algorithms and computer graphics. My goal is to focus my career on computer graphics. 
The Bead and Ring project at MDAnalysis resonated with my interest, and I hope I get to contribute to it.

**Comment by RMeli** [comment]
Thank you for participating in GSOC 2024 with us. Whether you were selected as a GSoC contributor or not, you are a valued member of the MDAnalysis community and we are excited to work with you outside of the GSoC program as well!
